define/fsd.md

```diff
@@ -190,7 +190,7 @@ However, when writing JSON, conforming clients and servers must always use [stan
 
 #### Validation
 
-Validation requirements can be specified with `[validate]`. This attribute results in invalid requests if decorated fields are unspecified in request parameters.
+Validation requirements can be specified with `[validate]`. This attribute results in invalid requests if decorated fields do not meet the validation requirements specified by the attribute. A validate method is generated on DTOs, which returns `true` if all criteria are met.
```

Comment:
> A validate method is generated on DTOs, which returns `true` if all criteria are met.

This might not be true for all supported languages. Probably best to let each language/framework document if/how it supports validation.

Reply:
I can also remove mention of request validation if you'd like. At some level that seems a primary and general use case to me, but maybe it isn't.